src/SourceBuild/tarball/content/repos/known-good.proj

```diff
@@ -43,7 +43,6 @@
 
         <!-- Tier 2 -->
         <RepositoryReference Include="linker" />
-        <RepositoryReference Include="runtime-portable" />
```

Comment:
The `runtime-portable.csproj` should get removed.  Also because of this change, I think it would be best to copy the the contents of runtime.common.props and runtime.common.targets into the runtime.proj file and delete these.  Having them separate provides no value and arguable hurts the readability of the code IMO.